src/pages/settings/Report/RoomNamePage.js

```diff
@@ -67,26 +77,28 @@ function RoomNamePage(props) {
 
     return (
         <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-            <HeaderWithBackButton
-                title={translate('newRoomPage.roomName')}
-                onBackButtonPress={() => Navigation.goBack(ROUTES.getReportSettingsRoute(report.reportID))}
-            />
-            <Form
-                style={[styles.flexGrow1, styles.ph5]}
-                formID={ONYXKEYS.FORMS.ROOM_NAME_FORM}
-                onSubmit={(values) => Report.updatePolicyRoomNameAndNavigate(report, values.roomName)}
-                validate={validate}
-                submitButtonText={translate('common.save')}
-                enabledWhenOffline
-            >
-                <View style={styles.mb4}>
-                    <RoomNameInput
-                        inputID="roomName"
-                        autoFocus
-                        defaultValue={report.reportName}
-                    />
-                </View>
-            </Form>
+            <FullPageNotFoundView shouldShow={Policy.isPolicyMember(policy)}>
```

Comment:
I was expecting the `shouldDisableRename(linkedWorkspace)` function itself to have gone into a util and we should have been using that util here to avoid duplicate logic.
So in future anyone who wants to change the permission, they can just update in the util and it will reflect in both the places.
@hungvu193

Reply:
I don't think so. It seems to be actually `!isPolicyAdmin()`, correct me if wrong. So It seems to be a duplicate.